### ISSUE TITLE
Fix Card Image

### ISSUE THREAD

**Issue by amanr-dev** [appwrite/pink#217]
The image is not showing in the Card UI example. Maybe the Image URL is broken.

### Screenshot :
![Screenshot 2024-08-19 131810](https://github.com/user-attachments/assets/1e766974-959c-49db-980c-31d432bbec29)

**Comment by nawfelsekrafi** [comment]
@amanr-dev I think the bot is talking about the screenshot image that u have included, it should have an alt text haha.

**Comment by sahilbhusari23** [comment]
Can I work on this ?
